### PR TITLE
chore: Fix repository location. Update generated docs after version bump.

### PR DESCRIPTION
**Pull request by themr0c**
> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?

* Fix repository location. 
* Update generated docs after version bump.

### What issues does this PR fix or reference?

* Fix builds

### Specify the version of the product this PR applies to.

7.31.x

### PR Checklist

As the author of this Pull Request I made sure that:

- [ ] `vale` has been run successfully against the PR branch
- [ ] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been configured on the docs side):
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

**Review by benoitf — APPROVED** [review]

File: modules/installation-guide/examples/system-variables.adoc [hunk 1]
```diff
@@ -102,13 +102,11 @@ pass:[<!-- vale off -->]
...
- `+CHE_INFRA_KUBERNETES_NAMESPACE_DEFAULT+`,"`+<username>-che+`","Defines Kubernetes default namespace in which user's workspaces are created if user does not override it. It's possible to use `<username>` and `<userid>` placeholders (for example: `che-workspace-<username>`). In that case, new namespace will be created for each user (or workspace). Used by OpenShift infra as well to specify a Project. The `<workspaceid>` placeholder is deprecated and is subject to removal in future releases. The `<username>` or `<userid>` placeholder is mandatory." 
+ `+CHE_INFRA_KUBERNETES_NAMESPACE_DEFAULT+`,"`+<username>-che+`","Defines Kubernetes default namespace in which user's workspaces are created if user does not override it. It's possible to use `<username>` and `<userid>` placeholders (for example: `che-workspace-<username>`). In that case, new namespace will be created for each user. Used by OpenShift infra as well to specify a Project. The `<username>` or `<userid>` placeholder is mandatory."
```
Comment (sleshchenko) [inline modules/installation-guide/examples/system-variables.adoc:106]: I suppose it's generated based on che.properties, but still, I'm not sure if

> if user does not override it.

makes sense anymore

**Review by sleshchenko — APPROVED** [review]
LGTM. Just one catch.

File: modules/installation-guide/examples/system-variables.adoc [hunk 2]
```diff
@@ -175,7 +172,7 @@ pass:[<!-- vale off -->]
...
+ `+CHE_WORKSPACE_STORAGE_AVAILABLE__TYPES+`,"`+persistent,ephemeral,async+`","The configuration property that defines available values for storage types that clients such as the Dashboard should propose to users during workspace creation and update. Available values:   - `persistent`: Persistent Storage slow I/O but persistent.   - `ephemeral`: Ephemeral Storage allows for faster I/O but may have limited storage       and is not persistent.   - `async`: Experimental feature: Asynchronous storage is combination of Ephemeral       and Persistent storage. Allows for faster I/O and keep your changes, will backup on stop       and restore on start workspace.       Will work only if:           - `che.infra.kubernetes.pvc.strategy='common'`           - `che.limits.user.workspaces.run.count=1`           - `che.infra.kubernetes.namespace.default` contains `<username>`      in other cases remove `async` from the list."
```
Comment (sunix) [inline modules/installation-guide/examples/system-variables.adoc:175]: the output format is strange here, is it intended ?
image

I would check with someone from platform team if it is ok for this part.

File: modules/installation-guide/examples/system-variables.adoc (hunk 1 above)
Comment (themr0c) [reply modules/installation-guide/examples/system-variables.adoc:106]: Then we need to address it in `che.properties`.
Can you please create an issue?

File: modules/installation-guide/examples/system-variables.adoc (hunk 2 above)
Comment (themr0c) [reply modules/installation-guide/examples/system-variables.adoc:175]: For the time being, any change in this file need to be addressed in the properties files.

We plan to change completely the output format. Tables are barely readable.

**Pull request merged**